ui/src/app/Routes.tsx

```diff
@@ -36,7 +36,7 @@ const Routes = (): JSX.Element => (
       )}
     />
     <Route
-      path={domainsURLs.domains}
+      path={[domainsURLs.domains, domainsURLs.details(null, true)]}
```

Comment:
`domainsURLs.details(null, true)` doesn't tell me what's happening here. I have to find the code where `details` is defined and inspect the function (and the resulting rabbit hole of the undocumented arguments of `argPath` ;) ). Instead of using positional arguments for boolean values it might be better for clarity to use an object with keys.

```js
domainsURLs.details({
  args: null,
  unmodified: true
})
```

Reply:
This is actually something Huw and I implemented just before the sprint. I've added an issue to address it separately.
https://github.com/canonical-web-and-design/maas-ui/issues/2756